fuzz.cpp

```diff
@@ -0,0 +1,64 @@
+#include <libriscv/machine.hpp>
```

Comment:
Was this written from scratch or copied from somewhere? `fuzz.cpp` is not a very descriptive filename.